package.json

```diff
@@ -105,15 +106,16 @@
     "karma-jasmine-html-reporter": "0.2.2",
     "karma-junit-reporter": "1.1.0",
     "karma-requirejs": "0.2.6",
-    "karma-selenium-webdriver-launcher": "0.0.4",
+    "karma-selenium-webdriver-launcher": "github:brian-smith-tcril/karma-selenium-webdriver-launcher#loosen-deps",
```

Comment:
Are we looking to ditch selenium altogether, or do we intend to fix it?

Reply:
I'm not sure. For now I just made it so we can use v3 https://github.com/PixnBits/karma-selenium-webdriver-launcher/commit/b8128678575687617f7ea9c6b40d66a333d12cc0

Reply:
I believe removing this is out of scope for this PR, but I made a DEPR ticket to move away from having this as a dependency https://github.com/openedx/public-engineering/issues/248